src/client/pages/ConnectPayment/index.tsx

```diff
@@ -20,7 +37,16 @@ export const ConnectPayment: React.FC = () => {
           <title>{textKeys.ONBOARDING_CONNECT_DD_PAGE_TITLE()}</title>
         </Helmet>
         <TopBar>
-          <LanguagePicker path="/new-member/connect-payment" />
+          {currentLocale.phoneNumber ? (
+            <PhoneNumber
+              color="white"
+              onClick={(status) =>
+                onClickPhoneTracking(history.location.pathname, status)
```

Comment:
nit: we don't need to pass `history.location.pathname` here. We could just pick that up in the `onClickPhoneTracking` function.

Reply:
👍